src/ontology/cl-edit.owl

```diff
@@ -30640,6 +30642,16 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:29904965"^^xsd:string) o
 AnnotationAssertion(rdfs:label obo:CL_4032001 "reelin GABAergic cortical interneuron")
 EquivalentClasses(obo:CL_4032001 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000013879)))
 
+# Class: obo:CL_4033013 (suprabasal keratinocyte)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:35883599"^^xsd:string) Annotation(oboInOwl:hasDbXref "https://www.proteinatlas.org/humanproteome/single+cell+type/squamous+epithelial+cells#suprabasalkeratinocytes"^^xsd:string) obo:IAO_0000115 obo:CL_4033013 "A keratinocyte of the epidermis suprabasal layer that has migrated from the basal stratum.")
```

Comment:
Both references describe this cell as differentiating (marker keratin 10). Consider including this in text definition. 

Is it possible that there is a cell type that is a keratinocyte of the epidermis suprabasal layer that has NOT migrated from the basal stratum? If not, consider whether that differentia is appropriate.

Reply:
Done. As the new model proposes that suprabasal keratinocytes are proliferative (although the native cell stems from the basal layer), it can be considered that keratinocytes do not migrate from the basal layer